homeworks/08-functional-reactive-programming/src/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { GRID_CELL_COUNT, INITIAL_WINDOWS } from './consts';
+import { Cell, CellContent } from './types';
+
+export const generateGrid = (): Cell[] => {
+  const cells: Cell[] = [];
```

Comment:
To avoid imperative code you can use `Array.from`.

Reply:
Done in https://github.com/vbarbasins/evo-ts-bootcamp/pull/9/commits/cf9028088f46d7fcd8728ac287f9e761297b979c